<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatAiWgt</class>
 <widget class="QWidget" name="ChatAiWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="AddPb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>开启新聊天</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="ChatAiLstWgt">
       <property name="styleSheet">
        <string notr="true">QListWidget#ChatAiLstWgt::item{
	color:black;
	background-color:rgb(203,203,195);
}
QListWidget#ChatAiLstWgt::item:selected {
	background-color:gray;
}
QListWidget#ChatAiLstWgt::item:hover{
	background-color:lightgray;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="ChatAiSWgt"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
